File: examples/file/read/node/readFile.js
const { FileService } = require("micro-js-client/file");

// Read a file by path
async function readFile() {
  let fileService = new FileService(process.env.MICRO_API_TOKEN);
  let rsp = await fileService.read({
    path: "/document/text-files/file.txt",
    project: "examples",
  });
  console.log(rsp);
}

readFile();

File: examples/go/file/read/readFile.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/file"
)

// Read a file by path
func ReadFile() {
	fileService := file.NewFileService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := fileService.Read(&file.ReadRequest{
		Path:    "/document/text-files/file.txt",
		Project: "examples",
	})
	fmt.Println(rsp, err)
}
